devel/gdb/Portfile

```diff
@@ -56,13 +55,21 @@ compiler.blacklist      {clang < 601}
 # compiler supports the standard - getting rid of old Apple GCC versions and the like?
 compiler.cxx_standard   2011
 
+configure.dir ${workpath}/build
```

Comment:
Can you explain why this directory is needed?

Reply:
GDB has taken to whining about in-tree builds, so to get around that I copied some code from GCC, which has had a similar issue for quite a while. All of the configure-related changes were lifted directly from GCC's portfile.

Reply:
Gotcha.